src/extension/background-script/actions/ln/index.js

```diff
@@ -1,6 +1,6 @@
 import PubSub from "pubsub-js";
 import state from "../../state";
-import sendPayment from "./sendPayment";
+import sendPayment, { weblnPay } from "./sendPayment";
```

Comment:
I'd move that in a different file. Then we keep the convention of one action per file? - does that make sense?